src/ServicePulse.Host/app/js/services/factory.rxjs.js

```diff
@@ -0,0 +1,8 @@
+; (function (window, angular, undefined) {
+    'use strict';
+
+    angular.module('sc')
+        .service('rx', function () {
```

Comment:
why not use `factory` ? For now it is so simple that it could be constant as @timbussmann mentions. But semantically I read it as factory. I wonder if we envision adding here more code. I am fine with either of the solutions